xbout/geometries.py

```diff
@@ -74,50 +79,77 @@ def wrapper(add_geometry_coords):
     return wrapper
 
 
+def _set_default_toroidal_coordinates(coordinates):
+    if coordinates is None:
+        coordinates = {}
+
+    # Replace any values that have not been passed in with defaults
+    coordinates['x'] = coordinates.get('x', 'psi')
+    coordinates['y'] = coordinates.get('y', 'theta')
+    coordinates['z'] = coordinates.get('z', 'phi')
+
+
 @register_geometry('toroidal')
-def add_toroidal_geometry_coords(ds):
+def add_toroidal_geometry_coords(ds, coordinates=None):
+
+    coordinates = _set_default_toroidal_coordinates(coordinates)
```

Comment:
@johnomotani We made a breaking mistake here - `_set_default_toroidal_coordinates()` has no return value, so `coordinates` is always `None` from here on. 

I think we should write a test for the desired behaviour, then fix it. At the moment because of this you can't load a grid file with `master`.